src/config.c

```diff
@@ -545,7 +555,7 @@ static RRStatus setRedisConfig(RedisModuleCtx *ctx, const char *param, const cha
     return ret;
 }
 
-static char *getRedisConfig(RedisModuleCtx *ctx, const char *name)
+char *getRedisConfig(RedisModuleCtx *ctx, const char *name)
```

Comment:
If I'm not mistaken, we don't need to export this function

Reply:
its now called by the callback for config changes, hence why its exported.  I guess I could put the callback in config.c but then its in a different place from the other callbacks.

Reply:
If I'm not mistaken, this is called from `updateTLSConfig()` only, this function is also inside config.c. So, `getRedisConfig()` might still be static. Just ignore my comment if this is not the case :)

Reply:
you were right I forgot I consolidated them into that function when I had them duplicated before and needed it exported.